WindowsStubs/WindowsStubsTest/WindowsStubsTest.cpp

```diff
@@ -12,9 +12,13 @@ int main()
 {
 	timespec ts;
 
-	clock_getres(CLOCK_MONOTONIC, &ts);
-	clock_gettime(CLOCK_MONOTONIC, &ts);
-	GetprocessPerformance();
+	//clock_getres(CLOCK_MONOTONIC, &ts);
+	//clock_gettime(CLOCK_MONOTONIC, &ts);
+	//GetprocessPerformance();
+
+//	TaskExecuteAndWait("c:\\python27\\python.exe");
+	std::string str = TaskExecuteAndWait("where python", true);
+	TaskExecuteAndWait(str, false);
```

Comment:
I just use this for sanity testing. Will remove it once we have unit tests for the library.